src/main/java/in/org/projecteka/hiu/consent/model/ConsentRequest.java

```diff
@@ -19,6 +20,6 @@
     private List<HIType> hiTypes;
     private Permission permission;
     private ConsentStatus status;
-    private String createdDate;
+    private Date createdDate;
```

Comment:
Please forgive me for  asking such naive questions, what are the goals for this story?

1. Make sure right datatype is used?
2. Expect the datetime without specific timezone from the user (UTC)?

if both are true, we should use a date type which is very explicit about that, for example https://www.joda.org/joda-time/apidocs/org/joda/time/LocalDateTime.html or if there are any inbuilt date types which does that.

Sorry If I'm wrong.

Reply:
Hey @ganesanarun, 

1. Make sure right datatype is used?
- Yes, that is one of goals for the story, although that is all on the developer, we aren't using anything to make sure all people comply. 
2. Expect the datetime without specific timezone from the user (UTC)?
- We are expecting a minimum - yyyy-mm-dd'T'HH:ss and max - yyyy-mm-dd'T'HH:ss.SSSZ as a string which we then convert to UTC by default.

We are currently using ZonedDateTime - A date-time with a time-zone.